src/docs/Contributing/Principles/ContributionJourney/index.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: 'Contribution journey'
+order: 1
+---
+
+Mozaic is a design system that wants to make designers and developers collaboration easy. When you contribute to Mozaic, you make the work you need, you don't wait for our team to make this work available for you.
+
+We identified **6 phases** in this contribution funnel. Let's briefly explain what every phase is about.
+
+# 1. KICK-OFF
+
+### ❓ What?
+The kick-off is the time where the designer and the developer sit alongside with Mozaic and LEROY MERLIN's teams. This time is very crucial because both teams will explain the duo what's expected from them. We like to do this event **in real life**.
+
+### 👩‍🚀 Who?
+- Mozaic's team
+- LMF Core team
+- The designer and developer duo
+
+### 🎬 How?
+The duo has to ask himself "Where does this pattern go?"
+What does the duo want from this pattern: they need to do a pooling on paper for the design of the pattern and have a small talk on what they want to achieve with the documentation. This will allow them to speak about technical feasibility.
```

Comment:
Maybe giving time to the designer to think about it and start working on! Then re meeting with the dev to let him give him a basic prototype. 
Last time it wasn't a success to get both starting in the same time

Reply:
yes the idea is to make the kick-off together and after this is:
- designer works | developer waits
- pooling
- developer works | designer waits
- pooling